azure-pipelines.yml

```diff
@@ -749,6 +759,13 @@ jobs:
       continueOnError: true
 
     # The binary is built from https://microsoft.visualstudio.com/Analog/_git/systems.rustyhmdkit
-    - script: '$(System.ArtifactsDirectory)/findconnectedport/linux/findconnectedport'
-      workingDirectory: '$(System.ArtifactsDirectory)/findconnectedport/linux/'
+#    - script: '$(System.ArtifactsDirectory)/findconnectedport/linux/findconnectedport'
+#      workingDirectory: '$(System.ArtifactsDirectory)/findconnectedport/linux/'
+#      displayName: 'Reset K4A Device'
+
+    - script: './x86_64-linux-clang-relwithdebinfo/bin/AzureKinectFirmwareTool -r'
```

Comment:
Why do we have a second reset after all of the tests have completed?

Reply:
The connex is actually powering off the hardware, so I assumed that at a min it was at least resetting the HW.

Reply:
It is removed now